settings.gradle

```diff
@@ -1 +1,7 @@
 include ':app'
+
+includeBuild('../NewPipeExtractor') {
+    dependencySubstitution {
+        substitute module('com.github.TeamNewPipe:NewPipeExtractor') with project(':extractor')
+    }
+}
```

Comment:
This should be used only for testing locally. When you publish the PR, you should put in [app/build.gradle](https://github.com/TeamNewPipe/NewPipe/blob/dev/app/build.gradle#L139) `implementation 'com.github.TeamNewPipe:NewPipeExtractor:48b397567813a8b78a21d47e830e879b49e7036c' //your last newpipe extractor commit`.